client/dashboard/me/preferences-login/index.tsx

```diff
@@ -0,0 +1,165 @@
+
+	// Fetch current login preferences
+	const { data: loginPrefs } = useSuspenseQuery( userPreferenceQuery( 'login-preferences' ) );
+
+	// Fetch user's sites
+	const { data: sites = [] } = useQuery(
+		sitesQuery( { site_visibility: 'visible', include_a8c_owned: false } )
+	);
+
+	// Initialize local state from server data
+	const [ primarySiteId, setPrimarySiteId ] = useState< string | undefined >(
+		loginPrefs?.primarySiteId
+	);
+	const [ landingPage, setLandingPage ] = useState< LandingPage >(
+		loginPrefs?.defaultLandingPage || 'primary-site-dashboard'
+	);
+
+	// Update local state when server data changes
+	useEffect( () => {
+		if ( loginPrefs?.primarySiteId ) {
+			setPrimarySiteId( loginPrefs.primarySiteId );
+		}
+		if ( loginPrefs?.defaultLandingPage ) {
+			setLandingPage( loginPrefs.defaultLandingPage );
+		}
+	}, [ loginPrefs ] );
+
+	// Set default primary site if none selected and sites are available
+	useEffect( () => {
+		if ( ! primarySiteId && sites.length > 0 ) {
+			setPrimarySiteId( sites[ 0 ].ID.toString() );
+		}
+	}, [ primarySiteId, sites ] );
+
+	// Update preferences mutation
+	const updatePreferences = useMutation( userPreferenceMutation( 'login-preferences' ) );
+
+	// Check if form has been modified
+	const isDirty = Boolean(
+		loginPrefs &&
+			( loginPrefs.primarySiteId !== primarySiteId ||
+				loginPrefs.defaultLandingPage !== landingPage )
+	);
+
+	// Save button state
+	const saveButtonState = useSaveButtonState( {
+		isSaving: updatePreferences.isPending,
+		isSuccess: updatePreferences.isSuccess,
+		isDirty,
+	} );
+
+	// Prepare site options for CustomSelectControl
+	const siteOptions: SiteOption[] = sites.map( ( site: Site ) => ( {
+		key: site.ID.toString(),
+		name: getSiteDisplayName( site ),
+		__experimentalHint: getSiteDisplayUrl( site ),
+	} ) );
+
+	const handleSave = async () => {
+		try {
+			await updatePreferences.mutateAsync( {
+				...loginPrefs,
+				primarySiteId,
+				defaultLandingPage: landingPage,
+			} );
+
+			createSuccessNotice( __( 'Login preferences saved successfully.' ) );
+		} catch ( error ) {
+			createErrorNotice( __( 'Failed to save login preferences. Please try again.' ) );
+		}
+	};
+
+	return (
+		<Card className="preferences-login-card">
+			<CardBody>
+				<VStack spacing={ 5 }>
+					<Text as="h3" weight={ 500 }>
+						{ __( 'Login preferences' ) }
+					</Text>
+
+					{ sites.length > 0 && (
+						<VStack>
+							<CustomSelectControl
+								label={ __( 'PRIMARY SITE' ) }
+								options={ siteOptions }
+								value={ siteOptions.find( ( option ) => option.key === primarySiteId ) }
+								onChange={ ( { selectedItem } ) => {
+									if ( selectedItem?.key ) {
+										setPrimarySiteId( selectedItem.key );
+									}
+								} }
+							/>
+							<Text variant="muted" style={ { fontSize: '13px', marginTop: '4px' } }>
+								{ __( "Choose the default site dashboard you'll see at login." ) }
```

Comment:
As per pgz0xU-1S-p2 

```suggestion
								{ __( 'Choose the default site dashboard you’ll see at login.' ) }
```

Reply:
I think we should create a lint rule for this 🥲